apps/docs/components/common/newsletter-signup.tsx

```diff
@@ -125,3 +128,13 @@ export const NewsletterSignup: FC<{ size?: 'small' | 'large' }> = ({ size = 'lar
 		</div>
 	)
 }
+
+// Sendgrid List
```

Comment:
better to put this is the 'submit-newsletter-signup' file if you wanted to keep this reference around